examples/fluids/qfunctions/setupgeo.h

```diff
@@ -77,6 +77,7 @@ CEED_QFUNCTION(Setup)(void *ctx, CeedInt Q, const CeedScalar *const *in, CeedSca
     const CeedScalar A33  = J11 * J22 - J12 * J21;
     const CeedScalar detJ = J11 * A11 + J21 * A12 + J31 * A13;
 
+    if (detJ <= 0) printf("%s: detJ %g\n", __func__, detJ);
```

Comment:
@jeremylt Do you have thoughts about how we should report such a thing to the user? This is especially noisy (will generate millions of lines on real problems) so not suitable for merging, but it would be good to have a clear message to the user about inverted elements (usually represents a mesh IO problem or issues [like this](https://gitlab.com/petsc/petsc/-/issues/1302) that I'm pretty sure we've stumbled into).

Reply:
Making a debug macro for qfunction use would be easy enough, but reporting only once would be harder. No obvious solution comes to mind

Reply:
We could make a mesh checking gallery QFunction that uses context to flag if the mesh has inverted elements and only report once